solvers/DPStokes/mobility.h

```diff
@@ -64,7 +64,7 @@ class DPStokes: public libmobility::Mobility{
       this->dppar.zmin -= this->dppar.w*h/2;
     }
     if(this->wallmode == "bottom"){
-      this->dppar.zmin -= this->dppar.w*h/2;
+      this->dppar.zmax += this->dppar.w*h/2;
     }
```

Comment:
@RaulPPelaez does this change look correct? In my new test, the self mobility didn't go to zero at the bottom wall without this change. I suspect it's supposed to add a buffer to the top boundary instead of the bottom wall since a comment in the area says to add a buffer for an open boundary.

Reply:
yes indeed

Reply:
yes indeed, good catch.